Editor/Modules/TextureModule.cs

```diff
@@ -0,0 +1,114 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine.Profiling;
+using Unity.ProjectAuditor.Editor;
+using UnityEditor;
+using UnityEngine;
+
+namespace Unity.ProjectAuditor.Editor.Modules
+{
+    public enum TextureProperties
+    {
+        Name,
+        Format,
+        Shape,
+        ReadWrite,
+        StreamingMipMaps,
+        MinMipMapLevel,
+        Resolution,
+        SizeOnDisk,
+        Path,
+        Num
+    }
+
+    class TextureModule : ProjectAuditorModule
+    {
+        public static string[] searchTheseFolders;                 // Manually Choose Folders to do the FindAssets search within.
+
+
+        static readonly IssueLayout k_IssueLayout = new IssueLayout
+        {
+            category = IssueCategory.Texture,
+            properties = new[]
+            {
+                // new PropertyDefinition { type = PropertyType.Description, name = "Texture Analysis", longName = "Results of Texture Analysis "},
+                new PropertyDefinition { type = PropertyTypeUtil.FromCustom(TextureProperties.Name), format = PropertyFormat.String, name = "Name", longName = "Show Texture Name" },
+                new PropertyDefinition { type = PropertyTypeUtil.FromCustom(TextureProperties.Format), format = PropertyFormat.String, name = "Format", longName = "Show Texture Format" },
+                new PropertyDefinition { type = PropertyTypeUtil.FromCustom(TextureProperties.Shape), format = PropertyFormat.String, name = "Shape", longName = "Show Texture Shape" },
+                new PropertyDefinition { type = PropertyTypeUtil.FromCustom(TextureProperties.ReadWrite), format = PropertyFormat.String, name = "Writeable", longName = "Show Texture ReadWrite" },
+                new PropertyDefinition { type = PropertyTypeUtil.FromCustom(TextureProperties.StreamingMipMaps), format = PropertyFormat.String, name = "StreamingMipMaps", longName = "Show Texture StreamingMipMaps" },
+                new PropertyDefinition { type = PropertyTypeUtil.FromCustom(TextureProperties.MinMipMapLevel), format = PropertyFormat.String, name = "MinMipMapLevel", longName = "Show Texture MinMipMapLevel" },
+                new PropertyDefinition { type = PropertyTypeUtil.FromCustom(TextureProperties.Resolution), format = PropertyFormat.String, name = "Resolution", longName = "Show Texture Resolution" },
+                new PropertyDefinition { type = PropertyTypeUtil.FromCustom(TextureProperties.SizeOnDisk), format = PropertyFormat.String, name = "SizeOnDisk", longName = "Show Texture SizeOnDisk" },
+                new PropertyDefinition { type = PropertyTypeUtil.FromCustom(TextureProperties.Path), format = PropertyFormat.String, name = "Location", longName = "Show Texture Location Path" }
+                //  new PropertyDefinition { type = PropertyType.Area, name = "Texture Recommendations", longName = "Recommendations for Optimizations for the Textures in the project." }
```

Comment:
this layout will not have any diagnostic, please remove, as well as comment

Reply:
Have removed the Resolution, SizeonDisk, and Path Properties.  Was this the intent of the comment/request?

Reply:
Pushed in [Tweaks to Texture Module] commit.